### DISCUSSION TITLE
help with installing starship on conemu WSL and bash

### DISCUSSION THREAD

**Discussion by gitreapur** [starship/starship#4951] [Q&A]
Has anyone got this to work?
I'm not certain where to put the fonts, I've tried installing them in windows 11, I've put them in ~/.local/share/fonts and also in ~/.fonts. I'm using the fira code fonts.

I have it working on an ubuntu laptop,  and even on windows 11 wsl using the ubuntu terminal that launches wsl.
I can't get it to work inside of a conemu terminal, any chars beyond basic ascii is weird symbols.
bash in conemu is being lauched like this:
set "PATH=%ConEmuBaseDirShort%\wsl;%PATH%" & %ConEmuBaseDirShort%\conemu-cyg-64.exe --wsl -cur_console:pm:/mnt -cur_console:t:"<ubuntu>"

Thanks for any help.

**Comment by chipbuster** [comment]
If things beyond ASCII don't work (even things like emoji) then most likely ConEmu itself isn't properly handling Unicode.

I'm not familiar with ConEmu myself, but they seem to have [a page for ideas on how to fix Unicode issues](https://conemu.github.io/en/UnicodeSupport.html), maybe one of those will help you?

**Comment by gitreapur** [comment]
Thank you for your reply. 
Sadly I've already tried their notes on that page and none of it seems to work.
I suspect there is a bunch of stuff I don't understand with how terminals work that I'm missing something, probably simple :)
